adding new updates to Getting Started (#22250)
microsoftgraph/microsoft-graph-docs: includes/data-connect-quickstart-06.md

This exercise describes how to set up your Azure resources to be able to connect Microsoft Graph Data Connect (Data Connect) to it. In this step, you can either choose Azure Synapse or Azure Data Factory to create a pipeline to extract the data from Microsoft 365 to the Azure Storage account using Data Connect.

We recommend that you use Azure Synapse because it has more in-built capabilities for data processing.

## Setting up your Azure resources

# [Create an Azure Synapse Pipeline](#tab/AzureSynapsePipeline)

1. Open a browser and go to your [Azure portal](https://portal.azure.com/#home).

> 1. Sign in to the portal using an account with an [Application Administrator](/azure/active-directory/roles/permissions-reference#application-administrator) or [Application Developer](/azure/active-directory/roles/permissions-reference#application-developer) role. Ensure that you are signed-in with your "developer" account that has privileges to create Azure resources within your subscription.

Reviewer: Instead of using "" please be specific. 
```suggestion
1. Sign in to the portal using an account with an [Application Administrator](/azure/active-directory/roles/permissions-reference#application-administrator) or [Application Developer](/azure/active-directory/roles/permissions-reference#application-developer) role. Ensure that you are signed-in with your developer account that has privileges to create Azure resources within your subscription.
```

This exercise describes how to set up your Azure resources to be able to connect Microsoft Graph Data Connect (Data Connect) to it. In this step, you can either choose Azure Synapse or Azure Data Factory to create a pipeline to extract the data from Microsoft 365 to the Azure Storage account using Data Connect.

> We recommend that you use Azure Synapse because it has more in-built capabilities for data processing.

Reviewer 2: @rimisra2 why in-built?

Author: @rimisra2 it's a better description of the capabilities that we dont want the documentation to detail since it's not the focus of this file. Let's keep as is from the original file.

    ![A screenshot of the resource page with Open Synapse Studio highlighted.](../concepts/images/azure-synapse-studio-tile.png)

> 1. By default, Azure Synapse Analytics uses an integration runtime that auto-resolves the region. In this tutorial, we recommend to the default auto-resolve in the developer tenant.

Reviewer: Did you mean:
```suggestion
1. By default, Azure Synapse Analytics uses an integration runtime that auto-resolves the region. In this tutorial, we recommend to use the default auto-resolve in the developer tenant.
```
?

Please review.

> 1. By default, Azure Synapse Analytics uses an integration runtime that auto-resolves the region. In this tutorial, we recommend to use the default auto-resolve in the developer tenant.

Reviewer 2: ```suggestion
1. By default, Azure Synapse Analytics uses an integration runtime that automatically resolves the region. In this tutorial, we recommend using the default auto-resolve option in the developer tenant.
```

    1. Switch to **Manage (toolbox icon) > Integration runtimes > New**.

    1. Select **Azure, Self-Hosted**, and choose **Continue**.

        ![A screenshot of the integration runtime setup with Azure, Self-Hosted highlighted.](../concepts/images/data-connect-synapse-IR.png)

> ig

Reviewer 2: ```suggestion
```

    ![A screenshot of the pipeline page with the bottom pane highlighted.](../concepts/images/data-connect-synapse-activity-run.png)

> 1. In the **Details** screen, look for the status of the pipeline activity as highlighted in the following image.The status should progress through Initializing, Consent Pending, Extracting Data, Persisting Data and Succeeded – no further action needed from you for this.  

Reviewer: ```suggestion
1. In the **Details** screen, look for the status of the pipeline activity as highlighted in the following image. The status should progress through Initializing, Consent Pending, Extracting Data, Persisting Data and Succeeded – no further action needed from you for this.  
```

1. Open a browser and go to your [Azure portal](https://portal.azure.com/).

> 1. Sign in using an account with [Application Administrator](/azure/active-directory/roles/permissions-reference#application-administrator) or [Application Developer](/azure/active-directory/roles/permissions-reference#application-developer) role to your Azure portal. Ensure that you are signed-in with your "developer" account that has privileges to create Azure resources within your subscription.

Reviewer 2: ```suggestion
1. Sign in to the Azure portal with an account that has either the [Application administrator](https://chat.openai.com/azure/active-directory/roles/permissions-reference#application-administrator) or [Application developer](https://chat.openai.com/azure/active-directory/roles/permissions-reference#application-developer) role. Make sure you are using your **developer** account, which has the necessary privileges to create Azure resources within your subscription.
```

Author: @Danielabom please remove the chat.openai.com links and revert to the links from the original file.

Please let me know once you've edited the suggest and I can commit it.

> 1. Sign in to the Azure portal with an account that has either the [Application administrator](/azure/active-directory/roles/permissions-reference#application-administrator) or [Application developer](/azure/active-directory/roles/permissions-reference#application-developer) role. Make sure you are using your **developer** account, which has the necessary privileges to create Azure resources within your subscription.

Reviewer 3: Not letting me comment on that line, but can we make it clear that "1. The request will be sent to the global admin to be approved. For the context of this tutorial, we recommend opening another tab with your admin priviledges enabled so you can approve the pipeline request." only applies to PAM flow?
